src/getHumanAge.test.js

```diff
@@ -7,4 +7,47 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  // з точки зору еквівалентних класів,
+  // цей і наступний тести збігаються,
+  // але я все одно вирішила відокремити цей випадок
+  test(`should return zeros if parametrs are zeros`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return zeros if parametrs < 15`, () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test(`if parametrs = 15`, () => {
```

Comment:
```suggestion
  test(`if parametrs = 15`, () => {
```

I'm not sure if I understand the expected result here. When your test fails you will have the following result - `if parameters = 15 - fail` check pls all your test names

Reply:
fixed